lib/rubocop/cli.rb

```diff
@@ -77,6 +77,8 @@ def handle_exiting_options
 
       puts RuboCop::Version.version(false) if @options[:version]
       puts RuboCop::Version.version(true) if @options[:verbose_version]
+
+      configuration
```

Comment:
Agree with you. It is because I prefered don't repeat execute code `@config_store.for(Dir.pwd).for_all_cops` twice. What would you do in my place?